modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -451,6 +552,60 @@ class NDB_Form_User_Accounts extends NDB_Form
         }
         $this->addSelect('CenterID', 'Site', $siteOptions);
 
+        if ($editor->hasPermission('examiner_multisite')) {
+            //get site aliases
+            $DB      =& Database::singleton();
+            $aliases = $DB->pselect("SELECT CenterID, Alias FROM psc ", array());
+
+            foreach ($aliases as $row) {
+                $groupA[] = $this->createCheckbox(
+                    'ex_'.$row['CenterID'],
+                    $row['Alias'],
+                    array()
+                );
+            }
+            $groupB[] = $this->createLabel(
+                "&nbsp;&nbsp;&nbsp;&nbsp; <b>Radiologist: </b>"
```

Comment:
HTML in PHP is done throughout this module (see 670-678) and maintained here for consistency of the template form

<label class="col-sm-2">
 +                    {$form.examiner_sites.label}
 +                </label>
 +                <div class="col-sm-10">
 +                    {$form.examiner_sites.html}
 +                </div>